### DISCUSSION TITLE
我在触发角那里加了一个快捷键win+L，但是无法实现锁屏，开启了管理员模式运行了

### DISCUSSION THREAD

**Discussion by Master-Panda** [shuax/MouseInc.Settings#418] [Q&A]
我在触发角那里加了一个快捷键win+L，但是无法实现锁屏，开启了管理员模式运行了

**Comment by shuax** [comment]
![image](https://user-images.githubusercontent.com/6940583/191146163-b94466bf-bd92-46c7-820d-dc521bbcbea1.png)
